doc/release-notes.md

```diff
@@ -45,15 +45,235 @@ Notable Changes
+
+Masternodes are "registered" by special transactions called ProTx, and removed only by spending the collateral.
+A ProTx either creates a 10000-PIV collateral as tx output, or includes a reference to an unspent 10000-PIV utxo on chain (and a proof of ownership).
+See PR [#2267](https://github.com/PIVX-Project/PIVX/pull/2267) for more information.
+
+Upgrade instructions: !TODO
+
+### New RPC commands
+
+* `protx_list`
+    ```  
+    protx_list (detailed wallet_only valid_only height)
+    
+    Lists all ProTxs.
+    
+    Arguments:
+    1. detailed       (bool, optional, default=true) Return detailed information about each protx.
+                          If set to false, return only the list of txids.
+    2. wallet_only    (bool, optional, default=false) If set to true, return only protx which involves
+                          keys from this wallet (collateral, owner, operator, voting, or payout addresses).
+    3. valid_only     (bool, optional, default=false) If set to true, return only ProTx which are active/valid
+                          at the height specified.
+    4. height         (numeric, optional) If height is not specified, it defaults to the current chain-tip.
+
+    Result:
+    [...]                         (list) List of protx txids or, if detailed=true, list of json objects.
+    ```
+
+* `protx_register`
+    ```
+    protx_register "collateralHash" collateralIndex "ipAndPort" "ownerAddress" "operatorPubKey" "votingAddress" "payoutAddress" (operatorReward "operatorPayoutAddress")
+    
+    Creates and sends a ProTx to the network. The collateral is specified through "collateralHash" and collateralIndex, and must be an unspent
+    transaction output spendable by this wallet. It must also not be used by any other masternode.
+    
+    Arguments:
+    1. "collateralHash"        (string, required) The collateral transaction hash.
+    2. collateralIndex         (numeric, required) The collateral transaction output index.
+    3. "ipAndPort"             (string, required) IP and port in the form "IP:PORT".
+                                  Must be unique on the network. Can be set to 0, which will require a ProUpServTx afterwards.
+    4. "ownerAddress"          (string, required) The PIVX address to use for payee updates and proposal voting.
+                                  The private key belonging to this address must be known in your wallet, in order to send updates.
+                                  The address must not be already registered, and must differ from the collateralAddress
+    5. "operatorPubKey"        (string, required) The operator BLS public key. The private BLS key does not have to be known.
+                                  It has to match the BLS private key which is later used when operating the masternode.
```

Comment:
we don't have BLS keys yet :p.

Reply:
ALL rpc helps listed here are already modified as per #2421.